### PR TITLE
Improve wording

### PR DESCRIPTION
**Pull request by JakeConnors376W**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Installation
 
-The supported and best way to install Homebrew is on the
-[homepage](https://brew.sh).
+Instructions for installing Homebrew are on the [homepage](https://brew.sh).
```
Comment (MikeMcQuaid) [inline docs/Installation.md]: Can we keep the word "supported" here somewhere? Thanks!

Comment (JakeConnors376W) [reply docs/Installation.md]: Would the current wording work? If not, how about "Instructions for a supported install of Homebrew are on the homepage?

Comment (MikeMcQuaid) [reply docs/Installation.md]: The latter sounds great ❤️

Comment (JakeConnors376W) [reply docs/Installation.md]: @MikeMcQuaid updated

**Comment by MikeMcQuaid** [comment]
Great work! 
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @JakeConnors376W!

**Pull request merged**